.travis.yml

```diff
@@ -12,13 +12,17 @@ env:
   matrix:
     # Ubuntu 14.04 versions
     - DISTRIB="conda" PYTHON_VERSION="$TRAVIS_PYTHON_VERSION"
-      NUMPY_VERSION="1.8.2" SCIPY_VERSION="0.14.0" 
-      SCIKIT_LEARN_VERSION="0.15.1" MATPLOTLIB_VERSION="1.4.0"
+      NUMPY_VERSION="1.9" SCIPY_VERSION="0.14.0" 
+      SCIKIT_LEARN_VERSION="0.16.1" MATPLOTLIB_VERSION="1.4.0"
```

Comment:
Why this change? Did you rebase on master after #185 was merged? ``scikit-learn`` 0.16 is not packaged in the latest Ubuntu.